### PR TITLE
Adding Huggingface test. Model: flax-latest-hf-bert

### PR DESCRIPTION
**Pull request by suexu1025**
Adding test for model of huggingface glue/bert on flax:

flax-latest-hf-bert-mnli-conv-v4-8-1vm
flax-latest-hf-bert-mnli-func-v4-8-1vm
flax-latest-hf-bert-mrpc-conv-v4-8-1vm
flax-latest-hf-bert-mrpc-func-v4-8-1vm

### CODE REVIEW

File: tests/jax/huggingface/glue.libsonnet [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    export GCS_BUCKET=$(MODEL_DIR)
+    export OUTPUT_DIR='./bert-glue'
+
+    python3 run_flax_glue.py --model_name_or_path bert-base-cased \
+      --output_dir ${OUTPUT_DIR} \
+      --logging_dir ${OUTPUT_DIR} \
+      --per_device_train_batch_size 4 \
+      %(extraFlags)s
+    gsutil -m cp -r ${OUTPUT_DIR} $(MODEL_DIR)
+  |||,
```
Comment (allenwang28) [inline tests/jax/huggingface/glue.libsonnet:93]: Could we utilize a template such as this? https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/jax/latest/hf-vit.libsonnet#L20-L45

The base HF setup is defined here: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/jax/common.libsonnet#L171-L199

File: tests/jax/huggingface/targets.jsonnet [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License,
+
+local text_classification = import 'glue.libsonnet';
+
+// Add new models here
+std.flattenArrays([
+  text_classification.configs,
+])
```
Comment (allenwang28) [inline tests/jax/huggingface/targets.jsonnet:20]: Could we move out of the huggingface folder and into `tests/jax/latest` for now?

I like the idea though, might ask to move it out in a future PR but I would need to think about this for a bit

File: tests/jax/huggingface/glue.libsonnet [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  },
+  local v4_8 = {
+    accelerator: tpus.v4_8,
+  },
+  configs: [
+    bert + mnli + convergence + v4_8 + timeouts.Hours(10),
+    bert + mrpc + convergence + v4_8 + timeouts.Hours(1),
+    bert + mnli + functional + v4_8,
+    bert + mrpc + functional + v4_8,
+  ],
```
[the remark this answers is not in the document]
Comment (allenwang28) [reply tests/jax/huggingface/glue.libsonnet:120]: might need to change line 95 to not depend on `common.tpuVmV4Base`.

**Review by allenwang28 — COMMENTED** [review]
Thanks Qinwen - looks great! Just had a few comments.

File: tests/jax/huggingface/glue.libsonnet (hunk 1 above)
Comment (suexu1025) [reply tests/jax/huggingface/glue.libsonnet]: moved the test to flax/latest

File: tests/jax/latest/hf-glue.libsonnet [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    accelerator: tpus.v4_8,
+  },
+
+  configs: [
+    hf_bert_common + mnli + convergence + v4 + v4_8 + timeouts.Hours(10),
+    hf_bert_common + mrpc + convergence + v4 + v4_8 + timeouts.Hours(1),
+    hf_bert_common + mnli + functional + v4 + v4_8,
+    hf_bert_common + mrpc + functional + v4 + v4_8,
+
+    hf_bert_common + mrpc + convergence + v3 + v3_8 + timeouts.Hours(2),
```
Comment (allenwang28) [inline tests/jax/latest/hf-glue.libsonnet:106]: Thanks! Maybe we can leave out the v2/v3 convergence tests, what do you think?

Comment (suexu1025) [reply tests/jax/latest/hf-glue.libsonnet]: sounds good. removed

**Review by allenwang28 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**